doc/reference/translations.md

```diff
@@ -46,6 +62,22 @@ will be rendered as:
 <input placeholder="bonjour" other="yes"/>
 ```
 
+and the following template:
+
+```xml
+<div t-translation-context="fr" title="hello">hello</div>
+<div>Are you sure?</div>
+<input t-translation-placeholder="pt" placeholder="hello" other="yes"/>
```

Comment:
incorrect

Reply:
why? It is placeholder is in TRANSLATABLE_ATTRS

Reply:
done